Packs/DeveloperTools/Integrations/XsoarServerPowershell/XsoarServerPowershell.yml

```diff
@@ -0,0 +1,39 @@
+category: Messaging
+commonfields:
+  id: XsoarServerPowershell
+  version: -1
+configuration: []
+description: This integration allows interacting with XSOAR server.
+display: XSOAR Server Powershell
```

Comment:
@GalRabin let's change the display name:
```suggestion
display: XSOAR Powershell Testing
```

Reply:
You will need to also update the display name in the release notes and detailed description.

Reply:
Changed in all required places.